static/app/gettingStartedDocs/node/gcpfunctions.tsx

```diff
@@ -17,7 +17,9 @@ import {getInstallConfig, getSdkInitSnippet} from 'sentry/utils/gettingStartedDo
 type Params = DocsParams;
 
 const getSdkSetupSnippet = (params: Params) => `
+// IMPORTANT: Make sure to import Sentry at the top of your file.
```

Comment:
```suggestion
// Make sure to initialize Sentry
```

Not sure about the "IMPORTANT" part, feels a bit "in your face" but maybe this is fine :D But we should reword this to say initialize instead of import!